Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1895,6 +1947,66 @@ def get_notable_sequence_event_types(client: Client, args: Dict[str, str]) -> Tu
     return human_readable, entry_context, sequence_event_types_raw_data
 
 
+def fetch_ir_as_incidents(client: Client, args: Dict[str, str]):
+    query = args.get('query')
+    incident_type = args.get('incident_type')
+    priority = args.get('priority')
+    status = args.get('status')
+
+    incidents = []
+
+    last_run = demisto.getLastRun()
+
+    raw_response = client.get_list_incidents(query, incident_type, priority, status, last_run=last_run)
+
+    for incident in raw_response['incidents']:
+        incidents.append(format_single_incident(incident))
+
+    if last_run and 'start_time' in last_run:
+        start_time = last_run.get('start_time')
+    else:
+        pass
+
+    next_run = datetime.now(tz=timezone.utc).strftime(DATE_FORMAT)
+    demisto.setLastRun({'time': next_run})
```

Comment:
I think that in case that we have incidents. next time should be based not the latest incident. and if there ar no incidents, it should not change

Reply:
fixed